### PR TITLE
Feature/improve api

### PR DESCRIPTION
**Pull request by T0ffik**

### CODE REVIEW

**Review by Veryyapeee — APPROVED** [review]

File: src/providers/ApiProvider.js [hunk 1]
```diff
@@ -15,18 +15,16 @@ class ApiProvider {
         return ApiService.request('GET', 'SearchTitle', name);
     }
 
-    /**
-     * @param {string} name
-     */
-    searchMovie(name) {
-        return ApiService.request('GET', 'SearchMovie', name);
+    commingSoon() {
```
Comment (maciejopalinski) [inline src/providers/ApiProvider.js]: typo: `comingSoon`

File: src/providers/ApiProvider.js [hunk 2]
```diff
@@ -15,18 +15,16 @@ class ApiProvider {
...
+        return ApiService.request('GET', 'CommingSoon', '');
```
Comment (maciejopalinski) [inline src/providers/ApiProvider.js]: typo: `ComingSoon`

**Review by maciejopalinski — DISMISSED** [review]
It doesn't work. Haven't you tested your solution before pushing and requesting a review?
Except that, it looks and works very good!

File: src/providers/ApiProvider.js (hunk 2 above)
Comment (T0ffik) [reply src/providers/ApiProvider.js]: Done!

File: src/providers/ApiProvider.js (hunk 1 above)
Comment (T0ffik) [reply src/providers/ApiProvider.js]: Done!

**Review by maciejopalinski — APPROVED** [review]
Looks good! :))

**Pull request merged**